contrib/storage-jdbc/src/main/java/org/apache/drill/exec/store/jdbc/JdbcRecordReader.java

```diff
@@ -210,8 +212,8 @@ public void setup(OperatorContext operatorContext, OutputMutator output) {
         String name = columns.get(i - 1).getRootSegmentPath();
         // column index in ResultSetMetaData starts from 1
         int jdbcType = meta.getColumnType(i);
-        int width = meta.getPrecision(i);
-        int scale = meta.getScale(i);
+        int width = Math.min(meta.getPrecision(i), DRILL_REL_DATATYPE_SYSTEM.getMaxNumericPrecision());
```

Comment:
The function of `min()` is to prevent the precision of the back-end database from exceeding the limit of Drill, so a selection is required, Is right?

Reply:
Very often the precision from the type is much less than the precision of the actual values, so with these changes, Drill will be able to process such values. But if the actual value also has large precision, the exception added below will be thrown, and as pointed in the error message, the user may use `round()` UDF to fix such a case.

Reply:
Nice idea. Is it possible to add the description of UDF (with the limitation of precision) in the [JDBC](https://github.com/apache/drill/blob/gh-pages/_docs/connect-a-data-source/plugins/080-rdbms-storage-plugin.md) docs?

Reply:
We already have `round()` UDF description at https://drill.apache.org/docs/math-and-trig/